plaso/parsers/__init__.py

```diff
@@ -33,6 +33,7 @@
 from plaso.parsers import opera
 from plaso.parsers import pe
 from plaso.parsers import plist
+from plaso.parsers import postgresql
```

Comment:
move this below pls_recall so it's in order

Reply:
Done